spec/spec_helper.rb

```diff
@@ -25,11 +25,16 @@
   end
 
   config.before do |example|
-    I18n.locale = :en
-    Globalize.set_fallbacks_to_all_available_locales
     Setting["feature.user.skip_verification"] = nil
   end
 
+  config.around do |example|
+    I18n.with_locale(:en) do
+      Globalize.set_fallbacks_to_all_available_locales
```

Comment:
Because in the `I18n.locale = :en` was executed just before that line. I just wanted to keep the same execution order. 🤔 Should we keep that line in the `config.before` hook?

Reply:
According to the [RSpec documentation about `around` hooks](https://relishapp.com/rspec/rspec-core/v/3-10/docs/hooks/around-hooks):

> WARNING: around hooks will execute before any before hooks, and after any after hooks regardless of the context they were defined in.

So I guess the order is the same :thinking:. Even if it weren't, it seems to me the `set_fallbacks_to_all_available_locales` does not depend on the current locale (only on the default one) :thinking:.

Reply:
Ok, I'll move it back to the before hook then.